File: aoBlogs2/Models/LinkAliasesModel.vb

Option Explicit On
Option Strict On

Imports System
Imports System.Collections.Generic
Imports System.Text
Imports Contensive.BaseClasses

Namespace Models
    Public Class LinkAliasesModel        '<------ set set model Name and everywhere that matches this string
        Inherits DbModel
        '
        '====================================================================================================
        '-- const
        Public Const contentName As String = "Link Aliases"      '<------ set content name
        Public Const contentTableName As String = "ccLinkAliases"   '<------ set to tablename for the primary content (used for cache names)
        Private Shadows Const contentDataSource As String = "default"             '<------ set to datasource if not default
        '
        '====================================================================================================
        ' -- instance properties
        'instancePropertiesGoHere
        Public Property Link As String
        Public Property PageID As Integer
        Public Property QueryStringSuffix As String
        '

    End Class
End Namespace

File: source/aoBlogs2/Models/Db/LinkAliasesModel.cs


namespace Contensive.Addons.Blog.Models {
    public class LinkAliasesModel : DbModel {        // <------ set set model Name and everywhere that matches this string
        // 
        // ====================================================================================================
        // -- const
        public const string contentName = "Link Aliases";      // <------ set content name
        public const string contentTableName = "ccLinkAliases";   // <------ set to tablename for the primary content (used for cache names)
        private  const string contentDataSource = "default";             // <------ set to datasource if not default
        // 
        // ====================================================================================================
        // -- instance properties
        // instancePropertiesGoHere
        public string Link { get; set; }
        public int PageID { get; set; }
        public string QueryStringSuffix { get; set; }
        // 

    }
}